src/super_gradients/training/utils/segmentation_utils.py

```diff
@@ -264,7 +264,7 @@ def __init__(self, crop_size: Union[float, Tuple, List], fill_mask: int = 0, fil
         :param fill_image: grey value to fill image padded background.
         """
         # CHECK IF CROP SIZE IS A ITERABLE OR SCALAR
-        self.crop_size = crop_size
+        self.crop_size = tuple(crop_size)
```

Comment:
Why do you need this? if crop size is a single integer as legacy used it will failed.
Transforming crop_size to tuple is already handled in self.check_valid_arguments().
Is it about PIL require only tuple type? If that the case check_valid_arguments() is the place to handle that, after crop_size is validated as iterable.

Reply:
removed. was a mistake